docs/pages/versions/v54.0.0/sdk/securestore.mdx

```diff
@@ -16,7 +16,7 @@ import {
 } from '~/ui/components/ConfigSection';
 import { SnackInline } from '~/ui/components/Snippet';
 
-`expo-secure-store` provides a way to encrypt and securely store key-value pairs locally on the device. Each Expo project has a separate storage system and has no access to the storage of other Expo projects.
+`expo-secure-store` provides a way to encrypt and securely store key-value pairs locally on the device. Each Expo project has a separate storage system and has no access to the storage of other Expo projects. This library requires a real device for testing since simulators do not require biometric authentication when retrieving secrets unlike real iOS devices.
```

Comment:
```suggestion
`expo-secure-store` provides a way to encrypt and securely store key-value pairs locally on the device. Each Expo project has a separate storage system and has no access to the storage of other Expo projects.
```
I don't like this comment as it feels a bit out of place. I feel like we should include it as a note for the `requireAuthentication` prop.